apps/ensrainbow/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@ensnode/ensrainbow",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "build": "tsc",
+    "start": "node dist/index.js",
+    "ingest": "node dist/ingest.js",
+    "docker:build": "docker build -t ensnode/ensrainbow .",
+    "docker:push": "docker push ensnode/ensrainbow"
+  },
+  "dependencies": {
+    "@hono/node-server": "^1.4.1",
+    "classic-level": "^1.4.1",
+    "hono": "^4.0.5",
+    "progress": "^2.0.3"
+  },
+  "devDependencies": {
+    "@types/node": "^20.11.19",
+    "@types/progress": "^2.0.7",
+    "typescript": "^5.3.3"
+  }
+} 
```

Comment:
apologies for the nitpick, but can you configure your editor to always add newlines at end of files?

Reply:
I have added `"files.insertFinalNewline": true` in vscode settings.